package.json

```diff
@@ -47,6 +47,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.0.0",
+    "@appium/base-driver": "^8.0.0-beta.6",
```

Comment:
the driver cannot depend on both base driver versions. If we add this then the previous one must be removed. Also switching to v8 would make the driver incompatible to Appium v1

Reply:
@mykola-mokhnach why can't the driver depend on both packages? from a node/npm/js perspective, these are just 2 packages. as long as we don't try and import the same thing from both packages we are fine. the point of doing this is to avoid having to duplicate code in the old basedriver package and the new basedriver package that will only ever be used in an appium 2.0 context anyway. once this driver only supports appium 2.0, then we can migrate all imports to the `@appium/base-driver` package. a TODO at the import site should be enough to explain this.